src/ui/ui-kit/FrameList/FrameListItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import cn from 'classnames';
+import type { LinkProps } from 'react-router-dom';
+import { UnstyledLink } from '../UnstyledLink';
+import { UnstyledAnchor } from '../UnstyledAnchor';
+import { UnstyledButton } from '../UnstyledButton';
+import * as styles from './styles.module.css';
+
+export const FrameListItem = React.forwardRef(
```

Comment:
I think in design we have a lot of variations of these, i.e. "hightlighed" item, item without frame, etc
Should we consider all those states for this abstraction?

Reply:
Not sure about this. May be next time. Or let's just postpone this pull request